biscuit-auth/src/format/convert.rs

```diff
@@ -113,9 +113,6 @@ pub fn proto_block_to_token_block(
 
     detected_schema_version.check_compatibility(version)?;
 
-    let scopes: Result<Vec<Scope>, _> =
-        input.scope.iter().map(proto_scope_to_token_scope).collect();
-
```

Comment:
this will require removing the `?` below, i think, since this expression shadows the previous one.